WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsFragment.java

```diff
@@ -648,9 +599,8 @@ private boolean handleFeatureAnnouncementClick() {
     }
 
     private boolean handleAppLocalePickerClick() {
-        // if per-app language preferences are enabled and the device is on API 33+, take the user to the
-        // system app settings to change the language
-        if (mIsPerAppLanguagePrefsEnabled && Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+        // if  the device is on API 33+, take the user to the system app settings to change the language
```

Comment:
Minor (🔍): There is an extra space (` `) in this comment here that could be deleted (between the `if` and `the` words).